skymp5-server/cpp/lib_espm/Combiner.h

```diff
@@ -102,17 +108,20 @@ class Combiner
 
 // Helpers/utilities
```

Comment:
Do not leave code under comment

Reply:
You mean that I should remove `FindInGroup` entirely? Won't it be useful?

Reply:
Uncomment then

Reply:
Actually, probably we'll have to rewrite that anyways... and it's probably better to just use some map in `pImpl` to achieve this task without iterating. But anyways, left this func for now. I'm uncertain of its' destiny...